spec/index.html

```diff
@@ -6251,6 +6251,130 @@ <h5>LANG</h5>
+                  <td><code>LANGDIR</code></td>
+                  <td><code>""</code></td>
+                  <td><code>"rtl"</code></td>
+                  <td>New function</td>
+                </tr>
+                <tr>
+                  <td><code>DATATYPE</code></td>
+                  <td><code>rdf:langString</code></td>
+                  <td><code>rdf:dirLangString</code></td>
+                  <td></td>
+                </tr>
+              </tbody>
+            </table>
+          </section>
+          <section id="func-islang">
+            <h5>isLANG</h5>
+            <pre class="prototype nohighlight"> <span class="return">xsd:string</span>  <span class="operator">isLANG</span> (<span class="type">RDF term</span> <span class="name">term</span>)
+            </pre>
+            <p>
+              The function <code>isLang</code> returns <code>true</code> if the
+              argument is a literal with a language
+              tag. Otherwise, the function returns false.
+            </p>
+            <p>If the argument is a lteral, the function is
+              equivalent to testing for the datatype of the
+              literal being either <code>rdf:langString</code>
+              or <code>rdf:dirLangString</code>.
+            </p>
+             <div class="result">
+               <table>
+                 <thead>
+                   <tr>
+                     <th>Expression</th>
+                     <th><code>Result</code></th>
+                   </tr>
+                 </thead>
+                 <tbody>
+                   <tr>
+                     <td><code>isLANG("abc"@en)</code></td>
+                     <td><code>true</code></td>
+                   </tr>
+                   <tr>
+                     <td><code>isLANG(1)</code></td>
+                     <td><code>false</code></td>
+                   </tr>
+                   <tr>
+                     <td><code>isLANG(&lt;http://example/&gt;)</code></td>
+                     <td><code>false</code></td>
+                   </tr>
+                   <tr>
+                     <td><code>isLANG("abc@en--ltrl"</code></td>
+                     <td><code>true</code></td>
+                   </tr>
+                   <tr>
+                     <td><code>isLANG("تصميم المواقع"@ar-ltr)</code></td>
+                     <td><code>true</code></td>
+                   </tr>
+                 </tbody>
+               </table>
+             </div>
+          </section>
+          <section id="func-langdir">
+            <h5>LANGDIR</h5>
+            <pre class="prototype nohighlight"> <span class="return">xsd:string</span>  <span class="operator">LANGDIR</span> (<span class="type"><span class="type literal">literal</span></span> <span class="name">ltrl</span>)
+            </pre>
+            <p>
+              If the argument is a literal, 
+              The function <code>LANGDIR</code> returns the
+              language direction of a literal with datatype
```

Comment:
"base direction" is the term from RDF Concepts. Maybe needs a reference?

```suggestion
              base direction of a literal with datatype
```